zap/src/main/java/org/zaproxy/zap/control/AddOnInstaller.java

```diff
@@ -96,15 +92,13 @@ private AddOnInstaller() {}
      *   <li>Files;
      *   <li>Extensions;
      *   <li>Active scanners;
-     *   <li>Passive scanners.
```

Comment:
Move the period up to the previous line?

Reply:
Removed the semicolons instead, not really needed.